subsys/bluetooth/mesh/light_ctrl_internal.h

```diff
@@ -36,6 +36,13 @@ prop_format_get(enum bt_mesh_light_ctrl_prop id)
 		return &bt_mesh_sensor_format_perceived_lightness;
 	case BT_MESH_LIGHT_CTRL_PROP_REG_ACCURACY:
 		return &bt_mesh_sensor_format_percentage_8;
+#if CONFIG_BT_MESH_LIGHT_CTRL_SRV_REG
```

Comment:
Should not be in #if. This function will eventually be used in client applications as well, they will need to be able to construct and send these values without enabling the regulator.